common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
if the Cmd username isn't present, returning bans where username == null is reasonable, if perhaps not exceptionally useful. Going with that is fine.

I don't really care about the behavior beyond two points:
- The server should handle it gracefully
- The field stays optional so we don't have to change the protocol later once we want better ban history searches